api_tests/lib/bitcoind_instance.ts

```diff
@@ -78,4 +64,31 @@ export class BitcoindInstance implements LedgerInstance {
     public getUsernamePassword() {
         return { username: this.username, password: this.password };
     }
+
+    public getDataDir() {
+        return this.dbDir.name;
+    }
+
+    private writeLogFile() {
+        const output = `regtest=1
+server=1
+printtoconsole=1
+bind=0.0.0.0:${this.p2pPort}
+rpcbind=0.0.0.0:${this.rpcPort}
+rpcallowip=0.0.0.0/0
+nodebug=1
+rest=1
+acceptnonstdtxn=0
+zmqpubrawblock=tcp://127.0.0.1:28332
```

Comment:
Why are these ports hardcoded?

Reply:
> 've some concerns that the functions `fund/connectLND/openChannel` are in the wrong place

Please check out [this commit](https://github.com/comit-network/comit-rs/pull/2020/commits/c87ed75a1e4ddb7083e0f0e12da5b2189f960fd8).  It moves all the lnd code out of `Actor` and into a class of its own.

Reply:
Moved to the constructor, as done for the other ports.